src/main/java/xyz/groundx/caver_ext_kas/kas/kip7/KIP7.java

```diff
@@ -0,0 +1,1459 @@
+     * @param amount Amount of tokens to create.
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Call mintAsync(String addressOrAlias, String to, BigInteger amount, ApiCallback<Kip7TransactionStatusResponse> callback) throws ApiException {
+        return mintAsync(addressOrAlias, to, Numeric.toHexStringWithPrefix(amount), callback);
+    }
+
+    /**
+     * Mints a new token for a given user. <br>
+     * POST /v1/contract/{contract-address-or-alias}/mint <br>
+     *
+     * <pre>Example :
+     * {@code
+     * String contractAlias = "";
+     * String to = "";
+     * String amount = "0x8ac7230489e80000"; // 10 * 10^18
+     *
+     * Kip7TransactionStatusResponse response = caver.kas.kip7.mint(contractAlias, to, amount);
+     * }
+     * </pre>
+     *
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param to Klaytn account address to receive the newly created token.
+     * @param amount Amount of tokens to create. (in hexadecimal with the 0x prefix)
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip7TransactionStatusResponse mint(String addressOrAlias, String to, String amount) throws ApiException {
+        MintKip7TokenRequest request = new MintKip7TokenRequest();
+        request.setTo(to);
+        request.setAmount(amount);
+
+        return kip7Api.mintToken(chainId, addressOrAlias, request);
+    }
+
+    /**
+     * Mints a new token for a given user asynchronously. <br>
+     * POST /v1/contract/{contract-address-or-alias}/mint <br>
+     *
+     * <pre>Example :
+     * {@code
+     * ApiCallback<Kip7TransactionStatusResponse> callback = new ApiCallback<Kip7TransactionStatusResponse>() {
+     *     ....implements callback method.
+     * }
+     *
+     * String contractAlias = "";
+     * String to = "";
+     * String amount = "0x8ac7230489e80000"; // 10 * 10^18
+     *
+     * caver.kas.kip7.mintAsync(contractAlias, to, amount, callback);
+     * }
+     * </pre>
+     *
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param to Klaytn account address to receive the newly created token.
+     * @param amount Amount of tokens to create. (in hexadecimal with the 0x prefix)
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call mintAsync(String addressOrAlias, String to, String amount, ApiCallback<Kip7TransactionStatusResponse> callback) throws ApiException {
+        MintKip7TokenRequest request = new MintKip7TokenRequest();
+        request.setTo(to);
+        request.setAmount(amount);
+
+        return kip7Api.mintTokenAsync(chainId, addressOrAlias, request, callback);
+    }
+
+    /**
+     * Burns tokens. <br>
+     * POST /v1/contract/{contract-address-or-alias}/burn
```

Comment:
```suggestion
     * POST /v1/contract/{contract-address-or-alias}/burn <br>
```

Reply:
@sirano11 
Hi sirano. I think we should make a convention about it.
Some lines have all the <br> tags, and some not on the line.

It seems good to set up a convention about it and follow it.

I will not leave suggestions for corrections in the rest of the similar parts.
We should think about it first I think 😃